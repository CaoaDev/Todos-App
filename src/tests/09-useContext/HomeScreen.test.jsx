import React from 'react';
import { mount } from 'enzyme';
import { HomeScreen } from '../../Components/09-useContext/HomeScreen';
import { UserContext } from '../../Components/09-useContext/UserContext';

describe('Pruebas en <HomeScreen />', () => {
    const user= {
        name:'Carlos A.',
        email:'caoa@dev.com'
    }
    test('Debe mostrarse correctamente', () => {
        const wrapper = mount(
            <UserContext.Provider value={ {
                user
            } }>
                <HomeScreen />
            </UserContext.Provider>
        );
        expect(wrapper).toMatchSnapshot(`<HomeScreen />`);
    })
});