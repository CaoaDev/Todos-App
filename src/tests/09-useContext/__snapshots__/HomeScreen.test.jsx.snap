// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen /> Debe mostrarse correctamente: <HomeScreen /> 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Carlos A.",
   "email": "caoa@dev.com"
}
  </pre>
</div>
`;
